### PR TITLE
Add reconnect

### PR DESCRIPTION
**Pull request by Taump**
Changes proposed in this pull request:
- automatically reconnect in the event of a WebSocket drop

### CODE REVIEW

File: src/wsclient.js [hunk 1]
```diff
@@ -16,38 +16,48 @@ const wait = (ws, cb) => {
...
-        this.open = true;
-      }
-    });
+      ws.addEventListener('close', () => {
+        this.open = false;
+        if (reconnect) {
+          this.ws = null;
+          setTimeout(() => {
+            this.connect();
+          }, 1000)
```
Comment (bonustrack) [inline src/wsclient.js]: What is the reason to wait 1sec here?

Comment (Taump) [reply src/wsclient.js]: To save user resources

Comment (bonustrack) [reply src/wsclient.js]: Do you see a scenario where the connect function would run in loop infinitely? If not i don't see the reason to wait.

Comment (bonustrack) [reply src/wsclient.js]: Ok after some research it seem "close" can fire multiple time from the same connection so setTimeout make sense.

**Pull request merged**